htdocs/fourn/commande/card.php

```diff
@@ -1361,7 +1385,7 @@
 									$lines[$i]->fk_product > 0 ? $lines[$i]->fk_product : 0,
 									$product_fourn_price_id,
 									$ref_supplier,
-									$lines[$i]->remise_percent,
+									0,     // $lines[$i]->remise_percent must not be carried over from client to vendor
```

Comment:
The supplier order can be created from a supplier proposal. In such case, we must keep all information of the proposal. So we must use here 0 only to create supplier order from a customer order.